cot/src/request/extractors.rs

```diff
@@ -595,8 +596,8 @@ mod tests {
 
     #[cot::test]
     async fn urls_extraction() {
-        async fn handler() -> cot::Result<Response> {
-            Ok(Response::new_html(cot::StatusCode::OK, Body::empty()))
+        async fn handler() -> cot::Result<Html> {
+            Ok(Html::new(""))
```

Comment:
```suggestion
        async fn handler() -> Html {
            Html::new("")
```

KISS :kissing_heart: